README.md

```diff
@@ -61,17 +61,7 @@ $ helm search bitnami
 
 ### Setup a Kubernetes Cluster
 
-The quickest way to setup a Kubernetes cluster is with [Google Container Engine](https://cloud.google.com/container-engine/) (GKE) using [these instructions](https://cloud.google.com/container-engine/docs/before-you-begin).
-
-Finish up by creating a cluster:
-
-```bash
-$ gcloud container clusters create my-cluster
-```
-
-The above command creates a new cluster named `my-cluster`. You can name the cluster according to your preferences.
-
-> For setting up Kubernetes on other cloud platforms or bare-metal servers refer to the Kubernetes [getting started guide](http://kubernetes.io/docs/getting-started-guides/).
+The quickest way to setup a Kubernetes cluster is with [Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/), [AWS Elastic Container Service](https://aws.amazon.com/eks/) or [Google Container Engine](https://cloud.google.com/container-engine/) using their respective quick-start guides. For setting up Kubernetes on other cloud platforms or bare-metal servers refer to the Kubernetes [getting started guide](http://kubernetes.io/docs/getting-started-guides/).
```

Comment:
Can you update the GKE reference to Google Kubernetes Engine and the link to https://cloud.google.com/kubernetes-engine/ please?